lib/index.js

```diff
@@ -41,12 +42,23 @@ Server.prototype._createServer = function() {
     return;
   }
 
+  // connections
+  self._connections = {};
+
   var server = net.createServer(function(connection) { //'connection' listener
     new Connection(connection, self, {
       version: self.protocolVersion,
       authKey: self.authKey,
       protocol: self.protocol
     });
+    // assign id
+    connection.id = self.nextConnectionId++;
```

Comment:
You should do that in the Connection constructor. We pass the server, so you can do something like this:
```
function Connection(...) {
  this.id = this.server.getNextId() // where getNextId is function() { return this.nextConnectionid++ }
}
```